docs/validator/validator-rejoin.md

```diff
@@ -0,0 +1,119 @@
+---
+sidebar_position: 3
+title: Rejoin after Hard Fork
+slug: /validator-rejoin
+---
+
+# Rejoin as a Validator after the Fork
+
+In order for Lava Validators from `lava-testnet-1` to join `lava-testnet-2`, it is necessary to make some slight adjustments. This page is a guide for Validators who were participating in the network before the hard fork.
+
+## 📋 Prerequisites
+
+- 🔺 Upgrade to Go version [v1.20.5](https://go.dev/dl/go1.20.5.linux-amd64.tar.gz)
+
+
+## 📝 Written Guide (~20m)
+
+
+### 🗝️ Backup keys 
+
+:::tip
+Backing up keys is not strictly mandatory to rejoin, but it is recommended to prevent any loss.
+:::
+
+🔑 Steps:
+* Backup the private validator key file under `~/.lava/config` (`priv_validator_key.json`)
+* Backup the node key under `~/.lava/config` (`node_key.json`)
+* Backup account keys using the following commands: 
+```bash
+    ACCOUNT_NAME=?
+    STRONG_PASSWORD=?
+    lavad keys export $ACCOUNT_NAME --keyring-backend test <<< "$STRONG_PASSWORD" > $ACCOUNT_NAME.key
+```
+<br />
+
+### ♻️ Reset your node
+
+Reset the data folder by running the following command:
+
+```bash
+lavad tendermint unsafe-reset-all
+```
+
+<br />
+
+### 📥 Get new Genesis file
+
+Download the new genesis file into `~/.lava/config`. The file can be downloaded from https://github.com/lavanet/lava-config/blob/main/testnet-2/genesis_json/genesis.json
+
+<br />
+
+
+### ⬇️ Download the new Binary Version
```

Comment:
we are missing the seed node

Reply:
Is this something you or @David-Melamed can provide?

Reply:
Is that what he added in #152 ?

Reply:
@Yaroms

Reply:
Added via assistance from @David-Melamed 
commit 6858820a2cf13b03f11ceefd35a2a046248f6688